### PR TITLE
Update/remove express

### PR DESCRIPTION
**Pull request by edgarsilva**

### CODE REVIEW

File: lib/api.js [hunk 1]
```diff
@@ -37,61 +35,15 @@ API.prototype.defaults = {
...
-    var title = this.express.get('title'),
-        str;
-
-    str = 'Listening ' + title + ' @' + this.host + ':' + this.port;
-
-    console.log('Cylon + MQTT is now publishing.');
-    console.log(str);
-
-  }.bind(this));
+  console.log('Cylon + MQTT is now publishing.');
```
Comment (deadprogram) [inline lib/api.js]: Shouldn't we use `Cylon.logger`? Also, a better message might be be "cylon-api-mqtt ready" or something like that.

Comment (edgarsilva) [inline lib/api.js]: The api does not include cylon, that is why I do not use it. I can certainly add a function that does the same or either remove the comment completely. What do you think?

Comment (edgarsilva) [inline lib/api.js]: I'll update the msg though.

Comment (deadprogram) [inline lib/api.js]: OK got it. Update the message then, please.

**Comment by deadprogram** [comment]
:scissors:

**Pull request merged**